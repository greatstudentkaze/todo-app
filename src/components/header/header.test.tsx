import { shallow, ShallowWrapper } from 'enzyme';
import toJson from 'enzyme-to-json';

import Header from './index';

describe('Header component', () => {
  let component: ShallowWrapper;

  beforeEach(() => {
    component = shallow(<Header />);
  });

  it('renders correctly', () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});
