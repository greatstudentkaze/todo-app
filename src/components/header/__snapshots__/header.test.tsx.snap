// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders correctly 1`] = `
<header
  className="page-header"
>
  <TaskSearchForm />
</header>
`;
